README.md

```diff
@@ -62,6 +62,7 @@ In addition to this curated list of middleware, we would like to specifically ca
 - [retry](https://github.com/lostisland/faraday-retry) - retry intermittent HTTP failures
 - [rashify](https://github.com/lostisland/faraday-rashify) - parse responses into a [Hashie::Mash::Rash](https://github.com/hashie/hashie)
 - [follow_redirects](https://github.com/tisba/faraday-follow-redirects) - follow HTTP 30X redirects
+- [xml_response](https://github.com/soberstadt/faraday-decode_xml) - decode XML responses
```

Comment:
This is cool, but, I wonder, why does its name seem to differ in the link?

Reply:
Yeah, that was bad copy/paste, sorry!